src/Common/Commands.Common.Authentication/Authentication/ProtectedFileTokenCache.cs

```diff
@@ -73,27 +73,10 @@ public ProtectedFileTokenCache(string cacheFile, IDataStore store = null)
 
         private void Initialize(string fileName)
         {
-            lock (fileLock)
-            {
-                if (_store.FileExists(fileName))
-                {
-                    var existingData = _store.ReadFileAsBytes(fileName);
-                    if (existingData != null)
-                    {
-                        try
-                        {
-                            Deserialize(ProtectedData.Unprotect(existingData, null, DataProtectionScope.CurrentUser));
-                        }
-                        catch (CryptographicException)
-                        {
-                            _store.DeleteFile(fileName);
-                        }
-                    }
-                }
-                
-                // Create the file to start with
-                _store.WriteFile(CacheFileName, ProtectedData.Protect(Serialize(), null, DataProtectionScope.CurrentUser));
-            }
+            ReadFileIntoCache(fileName);
+
```

Comment:
HasStateChanged=true;

Reply:
Nice catch on this one.  Gave comment to your requested changes. :)